doc/Architecture.md

```diff
@@ -110,3 +110,10 @@ registering `OpenHandler`s. You can have a look at the code
 Theia comes with an extensive list of contribution points already. A good way
 to see what contribution points exist is to do a find references on
 `bindContributionProvider`.
+
+## See also
+
+For a high level overview of Theia, see this document: 
+
+Multi-Language IDE Implemented in JS - Scope and Architecture
```

Comment:
I would suggest making the title of the document a link instead:

[Multi-Language IDE Implemented in JS - Scope and Architecture](https://docs.google.com/document/d/1aodR1LJEF_zu7xBis2MjpHRyv7JKJzW7EWI9XRYCt48)

    [Multi-Language IDE Implemented in JS - Scope and Architecture](https://docs.google.com/document/d/1aodR1LJEF_zu7xBis2MjpHRyv7JKJzW7EWI9XRYCt48)